Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM alpine:3.4
+FROM golang:1.11-alpine
 
 RUN apk update && apk upgrade && \
-     apk add ca-certificates && \
+     apk add ca-certificates \
+     xz-dev \
+     musl-dev \
+     gcc && \
      rm -rf /var/cache/apk/*
 
-RUN mkdir /etc/deployments
+RUN mkdir -p /go/src/github.com/mendersoftware/deployments
```

Comment:
I would keep the original steps regarding `/etc` and `config.yaml`, and add a step to delete the source at the end. The fact that we build it inside the Dockerfile is an implementation detail, and the source is useless in the final container.